webapp/security/models.py

```diff
@@ -77,3 +120,41 @@ class Release(Base):
     version = Column(String, unique=True)
     codename = Column(String, unique=True)
     lts = Column(Boolean)
+
+
+class CVEReference(Base):
+    __tablename__ = "cve_reference"
+
+    id = Column(Integer, primary_key=True)
+    uri = Column(String)
+
+
+class Bug(Base):
+    __tablename__ = "bug"
+
+    id = Column(Integer, primary_key=True)
+    uri = Column(String)
+
+
+class Package(Base):
+    __tablename__ = "package"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    source = Column(String)
+    launchpad = Column(String)
+    ubuntu = Column(String)
+    debian = Column(String)
+    type = Column(String)
+    releases_status = relationship(
+        "CVERelease", secondary=package_release_status
+    )
+
+
+class CVERelease(Base):
```

Comment:
We are calling this `CVERelease` but this seems then establishing a relationship with `Package` instead?

Reply:
So,`cve_release` has a relationship with `package_release_status`  which in turn has a relationship with `package` and this one has the relationship with `cve`
https://docs.google.com/drawings/d/1dsU5KT0iC3QSIFcE4xlf1q8SsgeAuyypkKpU7qVB0Ww/edit

Reply:
From that schema, "Release" looks way more like a "PackageRelease" than a "CVERelease"?
Also does one CVE only link to one package?

Reply:
Yeah, when I named it, I was thinking of making a difference between notices and CVEs. No, CVEs can have multiple packages